JWBestPracticeApps/AutoplayVideoFeed/FeedItemCell.swift

```diff
@@ -23,28 +24,42 @@ class FeedItemCell: UITableViewCell, JWPlayerDelegate {
         }
 
         didSet {
-            guard let playerView = player?.view else {
+            guard let player = player, let playerView = player.view else {
                 return
             }
-
+            
             containerView.addSubview(playerView)
             containerView.isUserInteractionEnabled = false
             playerView.constraintToSuperview()
+            
+            // To avoid lag in video load show the local thumbnail before the player starts playing
+            if player.position <= 0.0 {
```

Comment:
this is not the best check. With  DVR streams you might have a negative position. If you just want to make sure playback has not  started, you can check  the player's state. If it's Idle, then playback has not started. 

The best way to  know if playback has started is but listening to the player's events. You can do so by setting  a delegate or listening to the player's event notifications. In the SDK  see JWCallbackHandler.

Reply:
Thanks Karim, I will use the `player.state == idle` in order to keep it simple.